helper-cli/src/main/kotlin/utils/PathExcludeGenerator.kt

```diff
@@ -43,11 +43,11 @@ internal object PathExcludeGenerator {
         val dirsToExclude = mutableMapOf<File, PathExcludeReason>()
 
         dirs.forEach { dir ->
-            PATH_EXCLUDES_REASON_FOR_DIR_NAME.forEach { (pattern, reason) ->
-                if (FileMatcher.match(pattern, dir.name, ignoreCase = true)) {
-                    dirsToExclude += dir to reason
-                }
-            }
+            val (_, reason) = PATH_EXCLUDES_REASON_FOR_DIR_NAME.find { (pattern, _) ->
```

Comment:
Please simply squash this to the commit in this PR where you introduced this code.